REQ-016 PantallaDeCarga

LozanoAndersonTheStain opened the pull request:
Se realizo pantalla de carga para las locaciones donde sea necesaria, de ser necesario realizar cambios por favor avisarme, para de esta manera no generar conflictos en dado caso que yo le añada algo o lo elimine... Tambien modifique la parte del Layout donde estamos probando todas las pantallas que realizamos, puede que genere un conflicto, para proximas vistas no lo haré. 😎

LozanoAndersonTheStain committed:
The slider for the home page is made #REQ-004

LozanoAndersonTheStain committed:
Revert "The slider for the home page is made #REQ-004"

This reverts commit 73ff2cadf303c59ef0120c518d9ffa208573b9fb.

LozanoAndersonTheStain committed:
:art: The slider for the home page is made #REQ-004

LozanoAndersonTheStain committed:
:art: 'SliderHome'

LozanoAndersonTheStain committed:
Revert ":art: 'SliderHome'"

This reverts commit e6643a010272dadf47b2c675add87bcc154ef160.

LozanoAndersonTheStain committed:
Se inicia la creación de la pantalla de galeria

LozanoAndersonTheStain committed:
:memo: The layout of the grid elements is updated.

LozanoAndersonTheStain committed:
:rocket: Loading screen is made for locations where it is required

d4lion committed:
Merge branch 'integration' into features/REQ-016/PantallaDeCarga

d4lion commented:
Queda resuelto el conflicto, puede ser mergeada sin problemas sobre las integrations, debe volverse a implementar la el slider dentro del componente que lo necesita o verificar si el nuevo componente viene de #6

d4lion commented on src/layouts/Layout.astro:
```diff
@@ -18,6 +20,5 @@ import Slider from '@components/Layouts/Slider.astro';
   </head>
   <body>
     <NavBar />
-    <Slider />
```
No usar nada dentro de componentes que se reutilicen, como layouts, solo dentro de componentes que renderizan paginas.

d4lion reviewed (approved):
En general se ve vien, recuerda mandar un video o image donde se logre ver el componente para generar una idea de como es o que hace en dado caso sea funcional.

d4lion committed:
:fire: Fix: delete imports

Se eliminarón los imports de galerias y pantallas de carga para no generar problemas en componentes que usen el Layout
